sr_moveit_hand_config/package.xml

```diff
@@ -32,8 +32,9 @@
   <run_depend>trac_ik_kinematics_plugin</run_depend>
   <build_depend>trac_ik_lib</build_depend>
   <run_depend>trac_ik_lib</run_depend>
-
-
+  <build_depend>mongodb</build_depend>
+  <run_depend>mongodb</run_depend>
+  <build_depend>scons</build_depend>
```

Comment:
Do this package really depend on scons, or is this a trick to facilitate the installation of mongodb?
If so, please add a comment in the package.xml